solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/Settings/SettingsViewModel.swift

```diff
@@ -2,12 +2,16 @@
 
 import Foundation
 
-class SettingsViewModel {
+struct SettingsViewModel {
   let title: SettingsTitle
   let items: [String]
 
   init(title: SettingsTitle, items: [String]) {
     self.title = title
     self.items = items
   }
+
+  var titleForSection: String {
+    return self.title.value
+  }
```

Comment:
this is great! We should do something similar for the number of items.
Otherwise, if we use a collection that has not the `count` property, the ViewController will break